src/config.rs

```diff
@@ -0,0 +1,39 @@
+use std::fs;
+use std::path::PathBuf;
+
+use anyhow::Result;
+use std::io;
+
+use crate::path::fuelup_dir;
+
+pub struct Config {
+    toolchains_dir: PathBuf,
+}
+
+impl Config {
+    pub(crate) fn from_env() -> Result<Self> {
+        let fuelup_dir = fuelup_dir();
+
+        let toolchains_dir = fuelup_dir.join("toolchains");
```

Comment:
This could be a function (I think the `latest` toolchain PR might add one come to think of it).